openprescribing/frontend/management/commands/import_ppu_savings.py

```diff
@@ -290,8 +295,7 @@ def handle(self, *args, **options):
                     group_by=entity_type,
                     month=options['month'],
                     limit=options['limit'],
```

Comment:
`options['limit']` is going to be 0, because in `tasks.json`, the command is called with no arguments.

Is this intended?